docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -51,7 +51,7 @@ Ensure that your environment includes the following:
   [sign up page](https://goteleport.com/signup/) to begin your free trial.
 
 - The `tsh` client tool version >= (=cloud.version=). To download these tools,
-  visit the [Downloads](/docs/cloud/downloads) page.
+  visit the [Downloads](/docs/deploy-a-cluster/teleport-cloud/downloads/) page.
```

Comment:
We should use relative paths an MDX file for links, e.g., `../deploy-a-cluster/teleport-cloud/downloads.mdx`. That way the docs engine can automatically add the `/ver/<version>` URL path segment for the current version of the docs and any other logic for local docs links.

(Come to think of it, this isn't really written down anywhere, so I can add a linter rule for this.)

Reply:
@ptgott how would that work for the link in the include below? I would imagine that the include could be used anywhere, and so the correct relative path would change depending on the number of child directories the doc calling it is in.

Reply:
In includes, relative paths are from the include file. There's logic in the docs engine that resolves the relative paths to relative paths from the main docs page.

Reply:
Added an issue here to flag this in the linter: https://github.com/gravitational/docs/issues/156